cron/sqs_queue_status.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2014 Mozilla Corporation
+
+# kombu's support for SQS is buggy
+# so this version uses boto
+# to read an SQS queue and put events into elastic search
+# in the same manner as esworker_eventtask.py
+
+
+import json
+import os
+import sys
+import socket
+import time
+from configlib import getConfig, OptionParser
+from datetime import datetime
+from hashlib import md5
+import boto.sqs
+from boto.sqs.message import RawMessage
+import base64
+import kombu
+
+sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), '../lib'))
+from utilities.toUTC import toUTC
+from utilities.to_unicode import toUnicode
+from utilities.remove_at import removeAt
+from utilities.is_cef import isCEF
+from utilities.logger import logger, initLogger
+from elasticsearch_client import ElasticsearchClient, ElasticsearchBadServer, ElasticsearchInvalidIndex, ElasticsearchException
+
+def getDocID(sqsregionidentifier):
+    # create a hash to use as the ES doc id
+    # hostname plus salt as doctype.latest
+    hash = md5()
+    hash.update('{0}.mozdefhealth.latest'.format(sqsregionidentifier))
+    return hash.hexdigest()
+
+def getQueueSizes():
+    logger.debug('starting')
```

Comment:
Would it be useful for this to log something like "starting sqs_queue_status getQueueSizes"?

Reply:
I don't think we should continue using an arguably suboptimal style  just because it's standardized. Adding more information here will only serve as an improvement and a reminder that we should make similar changes elsewhere.  My experience, and it seems our team's experience, with delaying such changes for a future concerted effort is that this future effort is never made.